src/index.js

```diff
@@ -0,0 +1,17 @@
+import temp from './temp';
+
+fetch('https://api.giphy.com/v1/gifs/translate?api_key=DDO6wmRcjThYc6vRzRg6YFuHyCd0QOcR&s&s=Clouds', { mode: 'cors' })
+  .then((response) => response.json())
+  .then((response) => {
+    document.body.style.backgroundImage = `url(${response.data.images.original.url})`;
+    document.body.style.backgroundRepeat = 'no-repeat';
+    document.body.style.backgroundSize = 'cover';
```

Comment:
Let us add another block to capture failure, thus when the request fails a default gifs can be used. I think this would resolve the error(associated with gif) in the browser console.